ultravox/training/train.py

```diff
@@ -302,7 +304,7 @@ def main() -> None:
             num_procs=args.eval_num_procs,
             num_samples=args.eval_num_samples,
             max_new_tokens=args.eval_max_new_tokens,
-            verbose=True,
+            log_dir=wandb.run.dir if wandb.run is not None else str(args.logs_dir),
```

Comment:
I wanted to keep to `logs_dir`, but I couldn't find a way to force W&B to use this path. I didn't spend extra time on it either since we only use W&B at the moment.

Reply:
maybe simplify it to:

`log_dir=wandb.run.dir if wandb.run else str(args.logs_dir)`